server/controllers/allocationsController.ts

```diff
@@ -503,10 +507,14 @@ export default class AllocationsController {
       allocationNotesSensitive,
       isSPOOversightAccessed
     )
+
     req.session.allocationForm = {
       otherEmails,
       sendEmailCopyToAllocatingOfficer,
     }
+
+    req.session.confirmInstructionForm = null
```

Comment:
Just a thought we might alos need this if they go back to the allocations list screen without completing